SU2_CFD/include/solvers/CIncEulerSolver.hpp

```diff
@@ -118,6 +119,17 @@ class CIncEulerSolver : public CFVMFlowSolverBase<CIncEulerVariable, INCOMPRESSI
   template<ENUM_TIME_INT IntegrationType>
   void Explicit_Iteration(CGeometry *geometry, CSolver **solver_container, CConfig *config, unsigned short iRKStep);
 
+  /*!
+   * \brief Compute necessary quantities (massflow, integrated heatflux, avg density)
+   *        for streamwise periodic cases. Also sets new delta P for prescribed massflow.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] config - Definition of the particular problem.
+   * \param[in] iMesh - current mesh level for the multigrid.
+   */
+  void GetStreamwise_Periodic_Properties(const CGeometry *geometry,
```

Comment:
Why is this one in Euler and the other in NS?

Reply:
hmm, yeah, moved it to be a NS function